src/react/components/forms/IssueCredixPass.tsx

```diff
@@ -0,0 +1,103 @@
+import { Wallet } from "@project-serum/anchor";
```

Comment:
![image](https://user-images.githubusercontent.com/5576686/147454925-671bb06e-0e0f-4c0d-b9bf-9467711f8238.png)
UI buttons seems to be a bit off

Reply:
<img width="864" alt="Screenshot 2021-12-27 at 2 37 42 PM" src="https://user-images.githubusercontent.com/55653609/147455499-c1a919f4-f983-4962-8400-535ef55ca7cc.png">
Okie, I'll see if I can do something about it. works on mine : )

Reply:
I would also make like a tab or anything, no buttons, it feels a little weird https://mui.com/components/tabs/

Reply:
I think Maxim resolved it already 👍